x-pack/plugins/ml/public/application/model_management/model_actions.tsx

```diff
@@ -142,6 +142,7 @@ export function useModelActions({
         icon: 'visTable',
         type: 'icon',
         available: (item) => !!item.metadata?.analytics_config?.id,
+        enabled: (item) => item.origin_job_exists === true,
```

Comment:
Can you add a tooltip to show why the action is disabled.

Reply:
Added in 6d8a4e51ba0d9df76f917cf298b5332e58170320
<img width="503" alt="image" src="https://github.com/elastic/kibana/assets/6446462/b4b3b938-2738-488c-a476-6404c3698cf2">